pandas/tseries/index.py

```diff
@@ -814,8 +815,7 @@ def _add_offset(self, offset):
                           "or DatetimeIndex", PerformanceWarning)
             return self.astype('O') + offset
 
-    def _format_native_types(self, na_rep=u('NaT'),
-                             date_format=None, **kwargs):
+    def _format_native_types(self, na_rep='NaT', date_format=None, **kwargs):
```

Comment:
k thanks

I don't know if you added that test (below) which discovered this bug? (if not pls add with a nice comment)